packages/components/src/styles/components/layout/flex.scss

```diff
@@ -8,12 +8,16 @@
 //
 
 .hds-layout-flex {
+  // default gap values
+  --hds-layout-flex-row-gap: 0;
+  --hds-layout-flex-column-gap: 0;
```

Comment:
Specifying the values here essentially resets them for nested layouts so that the values aren't improperly inherited.

Reply:
will take a look, thanks

Reply:
Refactored.